Jenkinsfile

```diff
@@ -20,5 +22,42 @@ pipeline {
                 sh label: 'Running Packer build', script: 'packer build pipeline/packer/mhs.json'
             }
         }
+
+        stage('Deploy') {
+            steps {
+                dir('pipeline/terraform/test-environment') {
+                    sh label: 'Initialising Terraform', script: 'terraform init -input=false'
+                    sh label: 'Applying Terraform configuration', script: """
+                            terraform apply -auto-approve \
+                            --var cluster_id=${CLUSTER_ID} \
+                            -var ecs_address=${DOCKER_REPOSITORY} \
```

Comment:
Minor: Does it make any difference doing `-var` vs `--var`? Otherwise, for consistency (and nothing else), could just do them all in the same way (along with the `-auto-approve`).